assets/constants.ts

```diff
@@ -17,6 +17,12 @@ export = {
   // Whether to allow custom event reporting to Google Analytics.
   "CAN_SEND_ANALYTICS_EVENTS": false,
 
+  // The term 'staging' is used instead of the classroom url fragment field
+  // in the URL for topics that are not yet attached to a classroom.
+  "DUMMY_CLASSROOM_URL_FRAGMENT": "staging",
+
+  "DEFAULT_CLASSROOM": "math",
```

Comment:
Explain what this means (i.e. in what sense is it the default?). Also is it the "classroom name" or the "classroom url fragment"? A classroom is more like a domain object, not a string.